src/workerd/api/sockets.h

```diff
@@ -43,6 +43,10 @@ struct SocketOptions {
   JSG_MEMORY_INFO(SocketOptions) {
     tracker.trackField("secureTransport", secureTransport);
   }
```

Comment:
```suggestion
  };
```

Reply:
Added a semicolon in the latest force push [04dae05](https://github.com/cloudflare/workerd/pull/2244/commits/04dae0534cbb42c6ebb81c2e0a76c81d8a84705f)

I read the docs for the JSG_TS macros (whew), but didn't check what the surrounding ones do 😁

Reply:
Hmmm, seems like it's still failing, with my very limited knowledge about the macros, maybe putting the type override before the memory info one would help? Not sure if that would fit the overall code style either, but a quick ctrl+f showed that `JSG_MEMORY_INFO` is always last in a struct definition